superset/__init__.py

```diff
@@ -43,7 +43,7 @@
 cache = LocalProxy(lambda: cache_manager.cache)
 conf = LocalProxy(lambda: current_app.config)
 get_feature_flags = feature_flag_manager.get_feature_flags
-get_css_manifest_files = manifest_processor.get_css_manifest_files
+get_manifest_files = manifest_processor.get_manifest_files
```

Comment:
This is breaking change. Anyone using `import get_css_manifest_files from superset` should take note.

But I suspect it's not used anywhere except `viz.py` (code search on GitHub found no other references).

In the future, Superset modules should probably use `__all__` to specify what to export publicly, and internal modules should not `import ... from superset` root.